boards/stm32f723e-disco/include/periph_conf.h

```diff
@@ -35,7 +35,11 @@
 #include "periph_cpu.h"
 #include "clk_conf.h"
 #include "cfg_rtt_default.h"
+#if defined(MODULE_PERIPH_USBDEV_HS_UTMI)
+#include "cfg_usb_otg_hs_phy_utmi.h"
+#else
```

Comment:
Could you add a bit to the `doc.txt` for the `stm32f723e-disco` board that mentions adding the `periph_usbdev_hs_utmi` module toggles the USB peripheral from fs phy to the utmi hs phy?